include/openenclave/bits/result.h

```diff
@@ -259,6 +259,11 @@ typedef enum _oe_result
      */
     OE_INVALID_QE_IDENTITY_INFO,
 
+    /**
+     * The quote verification collateral is invalid.
+     */
+    OE_INVALID_COLLATERAL,
```

Comment:
again, recommend either adding SGX into the name if this is SGX-only, or making it be generic like OE_INVALID_ENDORSEMENT.  The key question to answer is whether we want a separate result value for every TEE kind, or if we want a generic value that can be reused across TEEs.

Reply:
Thanks, changed to OE_INVALID_ENDORSEMENT